### DISCUSSION TITLE
Pautas para a Assembleia Geral 2024

### DISCUSSION THREAD

**Discussion by tiidadavena** [apyb/comunidade#128] [Divulgação]
Olá, pythonistas! Como vocês estão?

Em Outubro de 2024, durante a Python Brasil teremos a Assembleia Geral.

Pythonistas associadas e não associadas à APyB são convidadas a participar para contribuir com temas, propostas e sugestões de diversas formas:

- Propostas de ferramentas 
- Documentações, revisões para a atuação das pessoas organizadoras de eventos
- Propostas para fortalecer as organizações de outros eventos

Você tem alguma ideia que possa contribuir e quer levar para ser debatido? Fique à vontade para contribuir! 

👋🏻 linktr.ee/apyb

**Comment by gabubellon** [comment]
Minha primeira dúvida para contribuir com as pautas é se a assembleia e votação serão hibridas novamente esse ano (presencial e virtual)

**Reply by tiidadavena** [reply]
Estamos alinhando o processo da infraestrutura online da Python Brasil, no momento a organização busca por uma pessoa para atuar como Kahuna do evento online.

**Reply by rougeth** [reply]
A APyB pode se responsabilizar pela assembleia híbrida caso o evento não seja híbrido?

**Reply by gabubellon** [reply]
Sim, minha pergunta é pertinente somente a Assembleia, não precisando de uma estrutura de transmissão tão complexa quanto a de um evento.
Se o fator for ter pessoas para ser suporte ou auxiliar nessa transmissão na parte virtual podem contar comigo

**Reply by tiidadavena** [reply]
Acreditamos que a APyB seja capaz de realizar a asssembleia de forma híbrida caso a conferência acabe não tendo interação online. 

Provavelmente precisaríamos de uma conta paga no StreamYard para transmissão online e do servidor do Discord.

Sobre uma ferramenta para uma possível votação - cidade-sede ou inclusão de tópico para debate na assembleia - talvez uma ferramenta de votação online ou até mesmo um formulário Google poderia suprir essa demanda.

**Comment by tiidadavena** [comment]
## [ESTATUTO]
A APyB necessita de mudança do escritório fiscal de Caxias do Sul. O endereço atual deve-se ao fato de um dos membros fundadores da associação, Dorneles Treméa, residir nessa cidade.

Entretanto, envio e recebimento de itens podem chegar no endereço dos parentes dele, causando desconforto.

Além disso, o um endereço novo pode melhorar o tempo de resposta para algumas movimentações mais específicas (como recebimento de valores do exterior).

**Reply by naanadr** [reply]
[Villa Urbana](https://villaurb.com/): Rua José Tovasi, N 417, Bairro Cruzeiro, Caxias do Sul, Rio Grande do Sul.

Issue: https://github.com/apyb/tarefas/issues/811

**Comment by sergiomario** [comment]
Eu queria propor de discutirmos em grupo como podemos mudar de vez o endereço da Associação, para tirarmos do local de fundação que é a casa onde o Dorneles morava.

Tem que ser outro endereço em Caxias do Sul mesmo, senão quebra todo o fluxo que já temos de geração de documentos com a prefeitura, banco etc.

Anos atrás eu tentei fazer isso mas inacreditavelmente em Caxias do Sul não tinha um Impact Hub da vida ou qualquer outro cowork que vendesse esse serviço de endereço fiscal que geralmente é super barato, coisa de pagar 100 reais por mês.

Minha ideia é ver se já tem esse tipo de serviço lá e a gente ajudar a diretoria no que for preciso pra gente resolver esse problema logo.

**Reply by naanadr** [reply]
Alguns locais que oferecem endereço comercial foram levantados nessa issue https://github.com/apyb/tarefas/issues/811, mas como a @tiidadavena mencionou aqui https://github.com/apyb/comunidade/discussions/128#discussioncomment-10501213 precisaremos mudar o estatuto para incluir o novo endereço 😄

**Reply by sergiomario** [reply]
Muito bom! Vi lá algumas opções, embora estão meio caras. Se mudar antes da assembleia já dá tempo de votar a alteração ainda esse ano. 🤩

**Comment by naanadr** [comment]
## [ESTATUTO]

O @sergiomario tinha apresentado no ano passado uma mudança no estatuto para oficializar a assembleia hibrida (https://github.com/pythonbrasil/wiki/pull/292) e como o discussion é o canal oficial para trazer pautas para a assembleia, segue a mudança proposta por ele anteriormente:

### Art. 12 º 

#### Conteúdo anterior
```
Art. 12º - A Assembléia Geral reunir-se-á ordinariamente uma vez por ano, de
preferência durante a realização de evento de porte nacional, para dar
conhecimento aos seus membros do relatório da Diretoria, já aprovado pelo
Conselho Deliberativo, e eleger, quando couber, uma nova Diretoria e membros do
Conselho Deliberativo, conforme dispõem os termos do Estatuto;
extraordinariamente, para fins especiais e expressos, sempre que convocada pelo
Conselho Deliberativo, pela Diretoria ou por 1/5 (um quinto) dos Associados
Plenos e Efetivos em gozo dos seus direitos.
```

#### Conteúdo novo
```
Art. 12º - A Assembléia Geral reunir-se-á ordinariamente uma vez por ano, de
preferência durante a realização de evento de porte nacional e de forma híbrida,
com pessoas participantes de forma online ou presentes fisicamente nas dependências
da assembleia, para dar conhecimento aos seus membros do relatório da Diretoria,
já aprovado pelo Conselho Deliberativo, e eleger, quando couber, uma nova
Diretoria e membros do Conselho Deliberativo, conforme dispõem os termos do
Estatuto; extraordinariamente, para fins especiais e expressos, sempre que
convocada pelo Conselho Deliberativo, pela Diretoria ou por 1/5 (um quinto) dos
Associados Plenos e Efetivos em gozo dos seus direitos.
```

### Art. 13º 

#### Conteúdo anterior
```
Art. 13º - Ressalvadas outras disposições estatutárias, devem ser observadas as
seguintes normas para que a Assembléia Geral se realize e delibere:

a. a Assembléia Geral deverá funcionar, em primeira convocação, com a presença
   mínima de 1/3 (um terço) dos Associados com direito a voto, e em segunda
   convocação, 30 (trinta) minutos depois, com qualquer quorum a partir de 11
   (onze) membros;
```

#### Conteúdo novo
```
Art. 13º - Ressalvadas outras disposições estatutárias, devem ser observadas as
seguintes normas para que a Assembléia Geral se realize e delibere:

a. a Assembléia Geral deverá funcionar, em primeira convocação, com a presença
   mínima de 1/3 (um terço) dos Associados com direito a voto, e em segunda
   convocação, 30 (trinta) minutos depois, com qualquer quorum a partir de 11
   (onze) membros. Essa verificação de presença deverá levar em conta as pessoas
   conectadas por meios digitais;
```

**Reply by cecivieira** [reply]
Acho que a proposta a ser levada poderia ser com a possibilidade de ter uma assembleia em versão virtual , híbrida ou presencial.
Assim, caso haja alguma edição do evento 100% remota, já temos prevista como válida uma assembleia 100% virtual.

**Reply by rougeth** [reply]
> Assim, caso haja alguma edição do evento 100% remota, já temos prevista como válida uma assembleia 100% virtual.

Eu entendi que a intenção da mudança é priorizar a assembleia em formato híbrido, e não apenas oficializar o formato. Inclusive já temos precedente, durante edições da Python Brasil em 2020 e 2021, tivemos assembleia (inclusive fui eleito em uma delas :D).

**Reply by cecivieira** [reply]
Pelo que entendi, para que a assembleia tenha efeito legal é necessário está previsto no estatuto. Esse é o motivo da alteração.

Até hoje não tivemos problema porque nenhuma instituição se deteve neste ponto. 

@rougeth

**Reply by tiidadavena** [reply]
Complementando a interpretação do estatuto no seu texto atual, ele não especifica o local e o formato das assembleias. Por isso, as assembleias conduzidas no formato extraordinário (online) possuem validade.

**Reply by naanadr** [reply]
> Acho que a proposta a ser levada poderia ser com a possibilidade de ter uma assembleia em versão virtual , híbrida ou presencial.
Assim, caso haja alguma edição do evento 100% remota, já temos prevista como válida uma assembleia 100% virtual.

@cecivieira o texto então seria algo do tipo (abaixo)? Caso não seja isso que você está pensando, por favor enviar aqui na thread a sugestão do texto, para podermos apresentar as alterações na assembleia e fazermos o momento mais célere possível.

### Art. 12 º 

Art. 12º - A Assembleia Geral reunir-se-á ordinariamente uma vez por ano, de preferência durante a realização de evento de porte nacional, de forma preferencialmente presencial, online ou hibrida, quando houver condições; com pessoas participantes de forma online ou presentes fisicamente nas dependências da assembleia. Nesse momento será dado conhecimento aos seus membros do relatório da Diretoria, já aprovado pelo Conselho Deliberativo, e elegerá, quando couber, uma nova Diretoria e membros do Conselho Deliberativo, conforme dispõem os termos do Estatuto; extraordinariamente, para fins especiais e expressos, sempre que convocada pelo Conselho Deliberativo, pela Diretoria ou por 1/5 (um quinto) dos Associados Plenos e Efetivos em gozo dos seus direitos.

### Art. 13º 

Art. 13º - Ressalvadas outras disposições estatutárias, devem ser observadas as seguintes normas para que a Assembleia Geral se realize e delibere:

a. A Assembleia Geral deverá funcionar, em primeira convocação, com a presença mínima de 1/3 (um terço) dos Associados com direito a voto, e em segunda convocação, 30 (trinta) minutos depois, com qualquer quorum a partir de 11 (onze) membros. Essa verificação de presença deverá considerar as pessoas presentes nas dependências da assembleia, caso ela ocorra presencialmente, e as conectadas por meios digitais, caso ela ocorra online ou de forma híbrida;

**Comment by ericof** [comment]
Eu gostaria de levantar a possibilidade de termos assembléias extraordinárias realizadas de maneira virtual.
Essa modalidade seria utilizada para votações que necessitem ocorrer no período entre duas assembléias ordinárias.
Um exemplo prático seria.uma alteração do estatuto que teve sua discussão iniciada durante uma assembleia anual, mas que por restrições de tempo não pode ser concluída e teria sua votação realizada apenas depois de 12 meses.
O mecanismo de assinatura da ata poderia ser o assinador do Gov.br, ou outra solução que apresente valor legal.

**Comment by itepifanio** [comment]
Gostaria de sugerir a possibilidade das comunidades receber patrocínios para vários eventos de uma vez.

No GruPy-RN a gente sempre planeja o ano seguinte com um número X de eventos. A gente gostaria de propor no plano a possibilidade das empresas terem sua marca em todos os meetups do ano através de uma única cota, mas a política da APyB de não permitir caixa de um evento ser transmitido para outro acaba limitando essa ideia.

Acho que um meio termo seria receber a cota anual e dividir ela entre os X códigos de eventos. Para o patrocinador seria emitido apenas uma nota, mas para organização interna da APyB ficaria fixo o valor por evento que não pode ser transmitido para o próximo. Existiria essa possibilidade?

**Reply by cecivieira** [reply]
Sim, @itepifanio existe essa possibilidade de receber patrocínio para várias edições de um mesmo evento. Para isso é necessário que a organização informe a APyB o montante que será destinado para cada edição.

Desde o ponto de vista de organizadora, sugiro que esse tipo de abordagem seja alinhada com a organização do evento seguinte para que ela se comprometa em dar a contrapartida prometida em contrato para a patrocinadora. Ou seja, é importante que a futura organização, que receberá o dinheiro, entenda da importância para a reputação da comunidade que essa contrapartida seja cumprida, pois caso contrário será difícil que a empresa considere estabelecer um novo vínculo com edições futuras.

**Comment by meu-bem** [comment]
Oi, pessoal! Tudo bem com vocês? Espero que o dia esteja sendo incrível!

Gostaria de sugerir que a APyB disponibilizasse uma plataforma própria para assinatura de documentos digitais. Abaixo estão os principais pontos que justificam essa sugestão:

* Assim como um email <alguma_coisa>@python.org.br transmite maior credibilidade aos patrocinadores e apoiadores, ter nossa própria plataforma de assinatura digital também reforça essa seriedade. Na posição de organizadores, acabamos passando uma imagem menos profissional quando perguntamos "A empresa de vocês utiliza alguma plataforma de assinatura? Poderia me enviar para que eu assine e retorne?". Isso pode dificultar a obtenção de apoios.
* Durante o evento, conseguimos captar recursos que poderiam, eventualmente, cobrir os custos de uma plataforma de assinatura através do email institucional. No entanto, no início da organização, a única certeza que temos é a realização do evento. Para kahunas menos experientes, contar com esse suporte seria um alívio, eliminando uma preocupação extra e fortalecendo a segurança das negociações.

Agradeço a atenção!

**Reply by biancarosa** [reply]
Acho que a gente poderia sempre usar gov.br

**Comment by biancarosa** [comment]
Não sei se faz sentido mas vou colocar aqui a ideia:

Gostaria propor a criação de um conselho de assessoria de eventos cujo foco seja suportar os eventos que utilizam a estrutura da APyB.

Não sou muito prescritiva aqui no que deve ser feito - mas penso que pós Python Brasil, seria interessante ter um trabalho contínuo ao longo do próximo ano pra documentar algumas dores e melhorar alguns processos. 

A ideia é criar um suporte contínuo para quem está organizando os eventos pra que o trabalho seja mais fácil ao longo dos anos. 

(E eu acho que o Big Kahuna de cada PyBR precisa fazer parte desse conselho durante o próximo ano, de forma a auxiliar no suporte da próxima Python Brasil sem necessariamente ajudar a organizar o evento novamente)

**Reply by ramiroluz** [reply]
Um conselho com todos os Big Kahunas numa dessas?

**Reply by ericof** [reply]
Não sei se todos as Big Kahunas, mas ao menos aquelas pessoas que ainda tenham pique para colaborar.
